logging/cluster-logging-deploying.adoc

```diff
@@ -1,21 +1,21 @@
 :context: cluster-logging-deploying
 [id="cluster-logging-deploying"]
-= Installing cluster logging
+= Installing OpenShift Logging
 include::modules/common-attributes.adoc[]
 
 toc::[]
 
 
-You can install cluster logging by deploying
-the Elasticsearch and Cluster Logging Operators. The Elasticsearch Operator
-creates and manages the Elasticsearch cluster used by cluster logging. 
-The Cluster Logging Operator creates and manages the components of the logging stack. 
+You can install OpenShift Logging by deploying
+the Elasticsearch and cluster logging operators. The Elasticsearch Operator
+creates and manages the Elasticsearch cluster used by OpenShift Logging. 
+The cluster logging operator creates and manages the components of the logging stack. 
```

Comment:
"Cluster Logging Operator"
https://github.com/openshift/cluster-logging-operator/

Reply:
> "Cluster Logging Operator"
> https://github.com/openshift/cluster-logging-operator/

Thanks. I've capitalized "Cluster Logging Operator" throughout the docs.